tests/util/verify.sh

```diff
@@ -280,30 +281,59 @@ __cluster_state() {
     # TEMP WORKAROUND, don't check istio-system
     kubectl get ns -o name | sed '/istio-system/d'
     kubectl get all --ignore-not-found -n default
-    kubectl get istiooperators --ignore-not-found -n default 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get destinationrules --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get envoyfilters --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get gateways --ignore-not-found -n default -n istio-system  2>&1| grep -v "error: the server doesn't have a resource type"
-    kubectl get serviceentries --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get sidecars --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get virtualservices --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get workloadentries --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get authorizationpolicies --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get peerauthentications --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
-    kubectl get requestauthentications --ignore-not-found -n default -n istio-system 2>&1 | grep -v "error: the server doesn't have a resource type"
+    kubectl get istiooperators --ignore-not-found -n default 2>&1
+    kubectl get destinationrules --ignore-not-found -n default -n istio-system 2>&1
+    kubectl get envoyfilters --ignore-not-found -n default -n istio-system 2>&1
+    kubectl get gateways --ignore-not-found -n default -n istio-system  2>&1
+    kubectl get serviceentries --ignore-not-found -n default -n istio-system 2>&1
+    kubectl get sidecars --ignore-not-found -n default -n istio-system 2>&1
+    kubectl get virtualservices --ignore-not-found -n default -n istio-system 2>&1
+    kubectl get workloadentries --ignore-not-found -n default -n istio-system 2>&1
+    kubectl get authorizationpolicies --ignore-not-found -n default -n istio-system 2>&1
+    kubectl get peerauthentications --ignore-not-found -n default -n istio-system 2>&1
+    kubectl get requestauthentications --ignore-not-found -n default -n istio-system 2>&1
 }
 
-__cluster_snapshot() {
-    __cluster_state > __cluster_snapshot.txt 2>&1
+__create_cluster_snapshots() {
```

Comment:
Can you also change the comment on line 268: __cluster_snapshot -> __create_cluster_snapshots